src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -8,21 +8,21 @@
   --calcite-accordion-item-spacing-unit: #{$baseline/3};
   --calcite-accordion-item-padding: var(--calcite-accordion-item-spacing-unit)
     var(--calcite-accordion-item-spacing-unit);
-  @include font-size(-3);
+  font-size: theme("fontSize.-2");
```

Comment:
I think using the `@apply` syntax is a bit more terse and expressive for stuff like this:

```
@apply text--2;
```

Reply:
Thanks @paulcpederson. I was doing a dumb earlier and was getting errorz. I'll do dat.